src/diffusers/pipeline_utils.py

```diff
@@ -117,6 +118,23 @@ class AudioPipelineOutput(BaseOutput):
     audios: np.ndarray
 
 
+def is_safetensors_compatible(info) -> bool:
+    filenames = set(sibling.rfilename for sibling in info.siblings)
+    pt_filenames = set(filename for filename in filenames if filename.endswith(".bin"))
+    is_safetensors_compatible = True
```

Comment:
Just thinking of an edge case. If we have a folder that has no PyTorch weights but only Flax weights and we want to load it into `DiffusionPipeline` with `from_pt=True` then `is_safetensors_compatible` would return true even though there are no `.safetensors` in the repo, but just because `pt_filenames` is an empty list. It's not really a problem right now because if there are no PyTorch weights we can safety add `*.bin` to the ignore list, but it might make the function a bit "incorrect". 

Could it maybe be slightly cleaner here to do:
```suggestion
    is_safetensors_compatible = any(file.endswith(".safetensors") for file in filenames)
```

This way we say the repo is compatible with safetensors as long as at least one safetensors file exists

Reply:
This is would also fail because `from_pt=True` would still fail since there are no pytorch models.

But the error would be much better with your fix. Good catch !